src/main/java/mekanism/common/content/tank/TankMultiblockData.java

```diff
@@ -121,10 +112,8 @@ public void writeUpdateTag(CompoundTag tag, HolderLookup.Provider provider) {
     private float getScale() {
         return switch (mergedTank.getCurrentType()) {
             case FLUID -> MekanismUtils.getScale(prevScale, getFluidTank());
-            case GAS -> MekanismUtils.getScale(prevScale, getGasTank());
-            case INFUSION -> MekanismUtils.getScale(prevScale, getInfusionTank());
-            case PIGMENT -> MekanismUtils.getScale(prevScale, getPigmentTank());
-            case SLURRY -> MekanismUtils.getScale(prevScale, getSlurryTank());
+            case CHEMICAL -> MekanismUtils.getScale(prevScale, getChemicalTank());
+            //todo shouldn't this use the lowest amount? - Thiakil
```

Comment:
Unsure? My initial thinking was that in general we display the larger number for chemicals in the GUI when the tank is empty. Though technically based on the implementation details of MekanismUtils#getScale, because we pass zero as the stored amount, the passed capacity is effectively ignored.

Reply:
> My initial thinking was that in general we display the larger number for chemicals in the GUI when the tank is empty

but why? its misleading - though I think the capacity should be the same or at least very similar, it causes confusion when they get _less_ than they bargained for